crypto/crypto/src/ed25519.rs

```diff
@@ -448,7 +448,10 @@ impl Signature for Ed25519Signature {
     /// Batch signature verification as described in the original EdDSA article
     /// by Bernstein et al. "High-speed high-security signatures". Current implementation works for
     /// signatures on the same message and it checks for malleability.
-    #[cfg(all(feature = "batch", not(feature = "vanilla")))] // see https://github.com/dalek-cryptography/ed25519-dalek/issues/126
+    #[cfg(all(
+        feature = "batch",
+        not(any(feature = "vanilla", feature = "vanilla-u32"))
+    ))] // see https://github.com/dalek-cryptography/ed25519-dalek/issues/126
```

Comment:
Ha! We can actually remove this check, the [issue](https://github.com/dalek-cryptography/ed25519-dalek/issues/126) was fixed by the recent 1.0.0 release and [libra-update to it](https://github.com/libra/libra/pull/5779).

Reply:
Is the check you want removed the entire thing or just the `not(any(feature = "vanilla", feature = "vanilla-u32"))` clause?